### DISCUSSION TITLE
Where do I report outdated packages?

### DISCUSSION THREAD

**Discussion by Cyberavater** [chocolatey/choco#2285] [General] [answered]
I see many packages don't get regularly updated, where should I report them?
-Android Studio
-Flutter
Are some of the popular ones.

**Comment by gep13** [comment, marked as answer]
The triage process for packages is documented here:

https://docs.chocolatey.org/en-us/community-repository/users/package-triage-process#package-questions-requests-updates-andor-fixes-for-community-feed-triage-process

Hopefully that will help answer your questions.
